geometry/geometry_tests/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ set(
   robust_test.cpp
   screen_test.cpp
   segments_intersect_test.cpp
+  segment2D_tests.cpp
```

Comment:
Note the file right above this one.

Reply:
As agreed, I've refactored the code: 
 - moved AreSegmentsIntersected() to segment2d.hpp;
 - moved all the tested from segments_intersect_test.cpp to segment2D_tests.cpp.